packages/server/src/__tests__/plugin/drainHttpServer/stoppable.test.ts

```diff
@@ -113,7 +112,7 @@ Object.keys(schemes).forEach((schemeName) => {
         const err = await a.failure(
           request(`${schemeName}://localhost:${p}`).agent(scheme.agent()),
         );
-        expect(err.message).toMatch(/ECONNREFUSED/);
+        expect(err.code).toMatch(/ECONNREFUSED/);
```

Comment:
`.message` is undefined in Node 20 but `.code` is defined across all of our supported versions.